src/mozanalysis/segments/desktop.py

```diff
@@ -6,13 +6,15 @@
 from mozanalysis.metrics import agg_any
 from mozanalysis.segments import Segment, SegmentDataSource
 
+#: SegmentDataSource: The clients_last_seen table.
 clients_last_seen = SegmentDataSource(
     name="clients_last_seen",
     from_expr="`moz-fx-data-shared-prod.telemetry.clients_last_seen`",
```

Comment:
At some point we should probably switch to `mozdata`, I guess

Reply:
No time like the present!